roles/openshift_health_checker/openshift_checks/docker_image_availability.py

```diff
@@ -102,20 +106,12 @@ def local_images(self, images, task_vars):
 
     def is_image_local(self, image, task_vars):
         result = self.module_executor("docker_image_facts", {"name": image}, task_vars)
-        if result.get("failed", False):
-            return False
-
         return bool(result.get("images", []))
 
     def known_docker_registries(self, task_vars):
         result = self.module_executor("docker_info", {}, task_vars)
 
-        if result.get("failed", False):
```

Comment:
Similar reason as the previous comment, we return an empty list if there is a failure below, so this seemed redundant to have